### PR TITLE
update ethabi-derive dependencies

### PR DESCRIPTION
**Pull request by debris**
syn && quote to their latest versions

### CODE REVIEW

File: derive/src/constructor.rs [hunk 1]
```diff
@@ -129,6 +130,6 @@ mod tests {
 			}
 		};
 
-		assert_eq!(expected, c.generate());
+		assert_eq!(expected.to_string(), c.generate().to_string());
```
Comment (debris) [inline derive/src/constructor.rs]: `TokenStream` does not implement `PartialEq`, so I convert it to string as a workaround

**Comment by debris** [comment]
Once merged, I'll release `ethabi-derive 6.0.2`

File: derive/src/constructor.rs (hunk 1 above)
Comment (dvdplm) [reply derive/src/constructor.rs]: Out of curiosity, is there (a good) reason why it doesn't?

Comment (debris) [reply derive/src/constructor.rs]: I guess someone just overlooked it during the refactor of `quote` and `proc-macro`

**Comment by debris** [comment]
`6.0.2` released

**Pull request merged**